src/runtime-rs/crates/resource/src/network/endpoint/endpoints_test.rs

```diff
@@ -22,6 +23,37 @@ mod tests {
         network_pair::{NetworkInterface, NetworkPair, TapInterface},
         utils::link::net_test_utils::delete_link,
     };
+    use hypervisor::{device::device_manager::DeviceManager, qemu::Qemu};
+    use kata_types::config::{QemuConfig, TomlConfig};
+
+    async fn get_device_manager() -> Result<Arc<RwLock<DeviceManager>>> {
+        let path = env!("CARGO_MANIFEST_DIR");
+        let path = Path::new(path)
+            .join("../../../libs/kata-types/tests/texture/configuration-anno-0.toml");
+        let content = fs::read_to_string(path).unwrap();
+        let hypervisor_name: &str = "qemu";
```

Comment:
Is type declaration required here？and what about converting it to a const?

Reply:
just for test case.